geometry_msgs/Vector3[] vertices
